corehq/motech/requests.py

```diff
@@ -292,3 +311,42 @@ def validate_user_input_url_for_repeaters(url, domain, src):
                 'reason': e.reason,
             })
             raise
+
+
+def apply_request_backoff(
+    max_retries=MAX_TRIES,
+    exponential_base=DEFAULT_BACKOFF_RETRY_EXPONENT_BASE,
+    exponential_period=DEFAULT_EXPONENTIAL_PERIOD
```

Comment:
Please feel free to comment on the naming of these two params; in my head it makes sense, but I'm not sure if it's immediately obvious to someone without context what they are and their use.